### PR TITLE
updated with Mendix SSO v3.0.0 module 

### PR DESCRIPTION
**Pull request by MxMurshed**
UIA-42 [MxSSO] Update templates with MendixSSO and Administration module and using the connector logic

We updated this template with MendixSSO v3.0.0, this update will be followed in later update with enabling MendixSSO by default.

### CODE REVIEW

File: resources/templates/login_automatic_redirect.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```
Comment (KevinVlaanderen) [inline resources/templates/login_automatic_redirect.html:1]: These resources shouldn't be added

File: theme-cache/web/theme.compiled.css [hunk 2]
```diff
@@ -13642,7 +13642,6 @@ body {
   position: absolute;
   top: 30px;
   right: 30px;
```
Comment (KevinVlaanderen) [inline theme-cache/web/theme.compiled.css:13644]: Should the theme-cache changes be committed?

File: themesource/atlas_core/native/core/helpers/switch.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { brand } from "../../variables";
```
Comment (KevinVlaanderen) [inline themesource/atlas_core/native/core/helpers/switch.js:1]: Why are these changes here?

**Review by KevinVlaanderen — CHANGES_REQUESTED** [review]
Why were the SprintrFeedback widgets changed?

File: themesource/atlas_core/native/core/helpers/switch.js (hunk 3 above)
Comment (MxMurshed) [reply themesource/atlas_core/native/core/helpers/switch.js:1]: because in the main branch of this repository, there's a missing file _\themesource\atlas_core\web\core\helpers\_slider-color-variant_ so I updated the template with the latest version of Atlas Core (v3.0.8)

File: theme-cache/web/theme.compiled.css (hunk 2 above)
Comment (MxMurshed) [reply theme-cache/web/theme.compiled.css:13644]: No, not required, I will remove this folder as well
